org.eclipse.zest.tests/src/org/eclipse/zest/tests/examples/GraphJFaceTests.java

```diff
@@ -193,14 +193,14 @@ public void testGraphJFaceSnippet6() {
 		assertEquals(graph.getNodes().size(), 3);
 
 		GraphContainer container = (GraphContainer) graph.getNodes().get(0);
-		assertNode(container.getNodes().get(0), "rock");
-		assertNode(container.getNodes().get(1), "paper");
-		assertNode(container.getNodes().get(2), "scissors");
+		assertNode(container.getNodes().get(0), "Rock");
+		assertNode(container.getNodes().get(1), "Paper");
+		assertNode(container.getNodes().get(2), "Scissors");
 		assertEquals(container.getNodes().size(), 3);
 
-		assertConnection(graph.getConnections().get(0), "rock", "paper");
+		assertConnection(graph.getConnections().get(0), "Rock", "Paper");
 		assertConnection(graph.getConnections().get(1), "Second", "Third");
-		assertConnection(graph.getConnections().get(2), "Second", "rock");
+		assertConnection(graph.getConnections().get(2), "Second", "Rock");
```

Comment:
It probably really doesn't matter for an example, whether the names are capitalized...